docs/index.html

```diff
@@ -0,0 +1,519 @@
+      }
+      .logo > img {
+        width: 100%;
+        max-height: 100px;
+        object-fit: contain;
+      }
+      nav {
+        text-align: center;
+        font-size: 1.25em;
+      }
+      nav > a {
+        display: inline-block;
+        padding: 15px 25px;
+        color: var(--black);
+        font-family: var(--serif);
+        font-weight: 500;
+        text-decoration: none;
+        transition: background 0.25s ease, opacity 0.25s ease;
+      }
+      nav:hover a,
+      nav:focus a {
+        opacity: 0.25;
+      }
+      nav > a:hover,
+      nav > a:focus {
+        opacity: 1;
+        background: var(--medium);
+      }
+      .link {
+        display: inline-block;
+        margin-right: 5px;
+        padding: 5px 10px;
+        text-decoration: none;
+        transition: background 0.25s ease, color 0.25s ease;
+      }
+      .link:hover,
+      .link:focus {
+        color: var(--black);
+        background: var(--medium);
+      }
+      .link:hover i,
+      .link:focus i {
+        color: var(--black);
+      }
+
+      /* for mobile screens */
+      @media only screen and (max-width: 860px) {
+        header > section {
+          padding-bottom: 20px;
+        }
+        .logo {
+          margin-bottom: 20px;
+        }
+        nav > a {
+          width: 100%;
+          padding: 10px 15px;
+        }
+      }
+
+      /* anchor styles */
+      .anchor {
+        margin-left: 10px;
+        opacity: 0;
+        transition: opacity 0.25s ease;
+      }
+      h2:hover .anchor,
+      .anchor:focus {
+        opacity: 1;
+        background: none;
+      }
+      .anchor i {
+        margin: 0;
+      }
+      @media (hover: none) {
+        .anchor {
+          opacity: 1;
+        }
+      }
+      [data-glow='true'] {
+        animation: highlight_glow 2s;
+      }
+      @keyframes highlight_glow {
+        0% {
+          background: none;
+        }
+        10% {
+          background: var(--light);
+        }
+        100% {
+          background: none;
+        }
+      }
+    </style>
+  </head>
+  <body>
+    <header>
+      <section>
+        <div class="section">
+          <div class="logo">
+            <img src="logo.svg" />
+          </div>
+          <nav>
+            <a href="#summary">Summary</a>
+            <a href="#citation">Citation</a>
+            <a href="#approach">Approach</a>
+            <a href="#resources">Resources</a>
+            <a href="#acknowledgements">Acknowledgements</a>
+          </nav>
+        </div>
+      </section>
+    </header>
+    <main>
+      <section id="summary">
+        <div class="section">
+          <h2>Summary</h2>
+          <p>
+            BioBombe represents an approach to optimize extraction of biological
+            representations in gene expression data. Named after the large
+            mechanical devices built by Alan Turing and other cryptologists
+            during World War II to decode secret messages, BioBombe is used to
```

Comment:
@gwaygenomics : this _still_ sounds like a method instead of a research project + resource & may make your reviewer still feel the same way.